numba/inline_closurecall.py

```diff
@@ -92,6 +96,38 @@ def reduce_func(f, A, v):
                             modified = True
                             # current block is modified, skip the rest
                             break
+                        # check for stencil call
+                        if (guard(find_callname, self.func_ir, expr)
```

Comment:
Will refactor this code. The thing with `inline_closurecall.py` is that it has to remove all closure related `make_function` before type inference, which would fail otherwise.